test/acceptance/features/bindAppToServiceUsingSecret.feature

```diff
@@ -10,8 +10,7 @@ Feature: Bind values from a secret referred in backing service resource
     @olm
     Scenario: Inject into app a key from a secret referred within service resource
         Binding definition is declared on service CRD.
-        Given OLM Operator "backend" is running
```

Comment:
if removed, then we should remove `@olm` tag as well?

Reply:
Yes.